README.md

```diff
@@ -1 +1,98 @@
-# Things3 Low-Level Python API to the Database
\ No newline at end of file
+Get your Things app data into a Python dict.
+
+## Install
+```python
+$ pip install things.py
+```
+
+## Run
+```pycon
+>>> import things
+>>> things.tasks()
+[{'uuid': '2Ukg8I2nLukhyEM7wYiBeb',
+  'type': 'task',
+  'title': 'Make reservation for dinner',
+  'project': {'title': 'Throw Birthday Party', 'uuid': 'bNj6TPdKYhY6fScvXWVRDX'},
+  ...},
+ {'uuid': 'RLZroza3jz0XPs3uAlynS7',
+  'type': 'task',
+  'title': 'Buy a whiteboard and accessories',
+  'project': {'title': 'Set Up Home Office', 'uuid': 'w8oSP1HjWstPin8RMaJOtB'},
+  'notes': "Something around 4' x 3' that's free-standing, two-sided, and magnetic.',
+  'tags': [],
+  'checklist': [
+      {'title': 'Cleaning Spray', 'status': 'done', ...},
+      {'title': 'Magnetic Eraser', 'status': 'open', ...},
+      {'title': 'Round magnets', 'status': 'open', ...}
+  ],
+  ...
+
+>>> things.areas()
+[{'uuid': 'hIo1FJlAYGKt1Yj38vzKc3',
+  'type': 'area',
+  'title': 'Family',
+  'tasks': [],
+  'projects': [
+    {'uuid': '2Ukg8I2nLukhyEM7wYiBeb',
+     'type': 'project',
+     'title': 'Vacation in Rome',
+     'tasks': [
+       {'uuid': 'jqBQAmQJOdnqVJRCuwaXcU',
+        'type': 'task',
+        'title': "Borrow Sarah's travel guide",
+        ...}],
+     'headings': [
+       {'uuid': 'gln1iLefjDXKkKIQLTqiWE',
+        'type': 'heading',
+        'title': 'Pack',
+        'tasks': [...]
+        ...},
+ ...
+
+>>> things.tags()
+[{'uuid': 'hIo1FJlAYGKt1Yj38vzKc3',
+  'type': 'tag',
+  'title': 'Home',
+  'shortcut': None},
+ {'uuid': 'hIo1FJlAYGKt1Yj38vzKc3',
+  'type': 'tag',
+  'title': 'Office',
+  'shortcut': None},
+ ...
+
+>>> things.get('hIo1FJlAYGKt1Yj38vzKc3')
```

Comment:
we'll need to think about how to filter - at least we need named parameters (e.g. id, status, title (with placeholders?), ...)

Reply:
All sounds good to me. One idea: For the first version one might focus on completeness of data and let Python (or the user) do the filtering rather than SQL.

Reply:
My database is huge ;) So let's have some basic filters in mind when designing the basic API.

Reply:
👍  Out of curiosity, how big is huge? :)

Reply:
@AlexanderWillner ping

Reply:
20+MB is size with 50k+ entries. Let see how this scales when putting every task in memory as a Python dict.

Reply:
👌